changelog.md

```diff
@@ -1,3 +1,11 @@
+TBD
+===
+
+Features:
+---------
+* Keep cursor for cli_helpers. (Thanks: [Dick Marinus])
```

Comment:
Can we change this to a more user-friendly change message? Maybe something like:

Internal Changes:
* Use less memory when formatting results for display.